pkgs/by-name/do/doge/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  fetchpatch,
+}:
+python3Packages.buildPythonPackage rec {
+  pname = "doge";
+  version = "3.6.0";
+
+  src = fetchFromGitHub {
+    owner = "Olivia5k";
+    repo = "doge";
+    rev = version;
+    hash = "sha256-72nRghD5k0ofrlvV3hEdyrr6uzV4+8At1bOCmRZTxhk=";
+  };
+
+  patches = [
+    # https://github.com/Olivia5k/doge/pull/66, adds a small doge
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/Olivia5k/doge/pull/66.patch";
```

Comment:
URLs with `pull` can be deleted easily. Use the URL below instead:

https://github.com/Olivia5k/doge/commit/14e3ccc0a3f1e91862492e20a34d008768a34039.patch